test/module/shared_model/validators/validators.hpp

```diff
@@ -36,6 +36,24 @@ namespace shared_model {
         return Answer();
       }
     };
+    /**
+   * Block validator which always returns answer with no errors.
+   */
+    class BlockAlwaysValidValidator {
+     public:
+      Answer validate(const interface::Block &) const {
+        return Answer();
+      }
+    };
+  /**
+* Proposal validator which always returns answer with no errors.
+*/
+  class ProposalAlwaysValidValidator {
```

Comment:
These are looks awful, may you replace it with smth like:
```cpp
template<typename Iface>
struct AlwaysValidValidator {
  Answer validate(const Iface&) const {
    return {};
  }
}
```

And replace with template version